### DISCUSSION TITLE
highway/path tagged abbandoned are visible on the map

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#17951] [Maps Related]
Path tagged abbandoned shouldn't be visible! (like openstreetmap do).
Yesterday I do a wrong decision for this and I had to go back 

The way are tagged like this:
~~~
abandoned:highway=path
name=Senter de la Stua
note:project_page=percorso impraticabile causa tempesta VAIA
~~~

OsmAnd
![abandoned_path](https://github.com/osmandapp/OsmAnd/assets/2643448/3dd75b67-3aac-451a-b17a-450ce3558639)

OpenStreetmap
![abandoned_path_osm](https://github.com/osmandapp/OsmAnd/assets/2643448/9eef18ac-e187-4bb2-8b0e-f870208fb340)

**Comment by DmitryAlexei** [comment]
It seems, that you have changed data in OSM yesterday (https://www.openstreetmap.org/way/806188723). But OsmAnd vector maps are updated every month in the early days of every month, so your changes are not on the map yet.
In case you enable live updates (https://osmand.net/docs/user/personal/maps#osmand-live available with PRO subscription), changes you have made will be visible.
According to OsmAnd rendering (https://raw.githubusercontent.com/osmandapp/OsmAnd-resources/master/rendering_styles/default.render.xml) abandoned:highway=path is visible only starting from 15 zoom level.  Some people may be interested in using such roads for outdoor activity.

**Reply by drazde** [reply]
Yes I chenge to avoid other people make the same mistake, the path I had reported isn't aviable in any case no one can use for bike activities
In openstreetmap.org it is not visible anyway.
Yes I have the subscription

I think I will change my default layout but I think it is a problem
